### ISSUE TITLE
Add "Receiver" Column to the Receive Screen

### ISSUE THREAD

**Issue by apbendi** [ScopeLift/umbra-protocol#231]
To help clarify how the stealth address mechanism works to our users, we want to expose a bit more of what's going on underneath the hood. One idea is to actually show the automatically generated stealth address that has received the funds (or holds the ability to withdraw them from the contract) on the receive screen.

Below is a rough mockup of what this can look like. (Please ignore fonts and spacings in the mockup).

<img width="929" alt="Screen Shot 2021-09-21 at 7 15 24 AM" src="https://user-images.githubusercontent.com/1479802/134163636-c4ee9528-0d46-454b-b880-53897533e61e.png">

Requirements:

- [ ] Change the name of the "From" column to "Sender"
- [ ] Add a column on the receive screen named "Receiver"
- [ ] Add an info icon next to "Receiver", when the user hovers/taps on the icon, they should see small modal with a short explanation (text below)

> The stealth address which received these funds. It looks like an empty address, and it can't be tied to your account, but only you have the ability to generate its private key.

- [ ] Show the truncated address of the receiver, which can be sourced from the [announcements](https://github.com/ScopeLift/umbra-protocol/blob/master/umbra-js/src/classes/Umbra.ts#L321)

**Comment by radchukd** [comment]
@mds1 
4. There's already an etherscan link in the date column, perhaps, I should add copying as in sender's?

**Comment by apbendi** [comment]
> Your screenshot shows the same address for all Receivers, but in reality each address in that column will be different (just wanted to make sure that was clear, let me know if I misunderstood)

Correct! Just a mockup.

> I'm not sold on the column name "Receiver", since my definition of the Receiver would be "the logged in user that received all transactions shown in the screenshot". I know the tooltip is intended to clear that up, but something like "Stealth Address" or "Stealth Address Recipient" would be more clear, though perhaps a bit wordy.

Yeah I was on the fence about this as well. I also considered "Stealth Receiver" Or "Stealth Address" but was worried it would be too long and look odd wrapped. Let's try it out and see what they look like.

> Should the tooltip also say "Learn More." as the last sentence with a link to https://app.umbra.cash/faq#receiving-funds ?

Great idea, let's include this @radchukd.

> Ah, you're right that'd be inconsistent with how the sender column behaves, so I like your suggestion of copying the stealth address when clicked

Good call @radchukd, I like this idea as well. Let's do it!
